src/components/CldPlaceholder/CldPlaceholder.vue

```diff
@@ -2,10 +2,11 @@
   <img v-if="!isImageLoaded()" class="cld-placeholder" :src="src" :width="this.width" :height="this.height" v-bind="nonCldAttrs"/>
 </template>
 <script>
+  import { COMPONENTS, PLACEHOLDER_OPTIONS } from '../../constants';
```

Comment:
Could we separate constants into two files, one for transformations and one for regular constants?

Reply:
I will do it in a separate PR, since this will cause conflicts in the next PR maybe.